asm/custom-functions/src/rando/mod.rs

```diff
@@ -0,0 +1,481 @@
+// Custom Rando Functions go here
+
+// IMPORTANT: when adding functions here that need to get called from the game,
+// add `#[no_mangle]` and add a .global *symbolname* to custom_funcs.asm
+
+use core::{
+    ffi::{c_char, c_ushort, c_void},
+    ptr, slice,
+};
+
+use cstr::cstr;
+
+use crate::{
+    game::{
+        actor, arc,
+        bird::AcOBird,
+        events::ActorEventFlowMgr,
+        file_manager,
+        flag_managers::*,
+        item,
+        message::{text_manager_set_num_args, text_manager_set_string_arg, FlowElement},
+        minigame::SpecialMinigameState,
+        player,
+        reloader::{self, Reloader},
+    },
+    system::button::*,
+};
+
+#[link_section = "data"]
+static mut IS_FILE_START: bool = false;
+
+#[link_section = "data"]
+#[no_mangle]
+static mut FORCE_MOGMA_CAVE_DIVE: bool = false;
+
+#[no_mangle]
+pub fn process_startflags() {
+    (unsafe { *file_manager::get_ptr() }).anticommit_flag = 1;
```

Comment:
```suggestion
    unsafe { (*file_manager::get_ptr()).anticommit_flag = 1 };
```
Otherwise rust just ignores this code, probably because it's UB somehow?